LICENSE.md

```diff
@@ -1,4 +1,4 @@
-Copyright 2020 Inrupt Inc.
+Copyright 2022 Inrupt Inc.
```

Comment:
I learned from Kyra that it's actually copyright from, not copyright on, so this old date is fine (though maybe we need to clarify with legal)